src/core/Notification/Notification.baseStyles.ts

```diff
@@ -3,63 +3,75 @@ import { SuomifiTheme } from '../theme';
 import { font, element } from '../theme/reset';
 
 export const baseStyles = (theme: SuomifiTheme) => css`
+  ${element(theme)}
+  ${font(theme)('bodyTextSmall')}
+  width: 100%;
+  box-shadow: ${theme.shadows.wideBoxShadow};
+  border-radius: 4px;
+  justify-content: space-between;
+  padding-bottom: 10px;
   &.fi-notification {
-    ${element(theme)}
-    ${font(theme)('bodyTextSmall')}
-    padding-bottom: 17px;
-    width: 100%;
-    box-shadow: ${theme.shadows.wideBoxShadow};
-    border-radius: 4px;
+    & .fi-notification_element-wrapper {
+      overflow: hidden;
+    }
 
     & .fi-notification_text-content-wrapper {
-      margin-top: 20px;
+      padding-top: 18px;
       padding-left: 20px;
+      flex-grow: 1;
       & .fi-notification_content {
         vertical-align: middle;
-        ${font(theme)('bodyTextSmall')}
+        & .fi-notification_contentWrapper {
+          padding-top: 3px;
+          ${font(theme)('bodyTextSmall')}
+        }
       }
     }
-    & .fi-notification_label {
+    & .fi-notification_heading {
       ${font(theme)('bodySemiBold')}
-      margin-top: -1px;
-      margin-bottom: ${theme.spacing.xxs};
+      margin: 1px 0 0 0;
     }
     & .fi-notification_action-element-wrapper {
-      margin: 10px 20px 5px 89px;
+      padding: 20px 26px 19px 87px;
       & .fi-button {
+        margin-top: ${theme.spacing.xs};
         margin-right: ${theme.spacing.s};
       }
+      & .fi-button:first-child {
+        margin-top: 0;
+      }
     }
     .fi-notification_icon-wrapper {
-      margin-top: ${theme.spacing.m};
+      padding-top: 21px;
       flex: 0;
       & .fi-notification_icon {
         height: 24px;
         width: 24px;
       }
     }
-    & .fi-notification_close-button-wrapper {
+    & .fi-notification_close-button {
+      ${font(theme)('bodyTextSmall')}
       flex: 1 0 auto;
-      display: flex;
       flex-wrap: nowrap;
+      display: flex;
       box-sizing: border-box;
-      margin-top: 14px;
+      margin-top: 11px;
       margin-right: -5px;
-      margin-bottom: ${theme.spacing.insetM};
-    }
-    & .fi-notification_style-wrapper {
-      padding: 0 43px 10px 43px;
-      display: flex;
-      align-items: flex-start;
-    }
-    & .fi-notification_close-button {
-      ${font(theme)('bodyTextSmall')}
-      height: 40px;
+      margin-bottom: 0;
+      max-width: 50%;
       min-width: 40px;
+      text-align: right;
       padding: 7px 8px;
       border: 1px solid transparent;
```

Comment:
Regarding the border: in alert the border reaches the edges of the parent element, but here there's a small margin. Is this something that has been discussed with design or should the styles be aligned?

Reply:
Looks like this is because of the hidden overflow. Now the focus indicator fits inside the element too and is visible in its entirety.